[WARNING] 2025-10-26 22:15:53,757 | django.server | "GET /static/frontend/css/style.css HTTP/1.1" 404 1880
[WARNING] 2025-10-26 22:15:55,217 | django.request | Not Found: /favicon.ico
[WARNING] 2025-10-26 22:15:55,220 | django.server | "GET /favicon.ico HTTP/1.1" 404 6781
[WARNING] 2025-10-26 22:15:58,789 | django.request | Not Found: /accounts/login/
[WARNING] 2025-10-26 22:15:58,790 | django.server | "GET /accounts/login/?next=/dashboard/student/ HTTP/1.1" 404 6818
[WARNING] 2025-10-26 22:16:03,231 | django.request | Not Found: /accounts/login/
[WARNING] 2025-10-26 22:16:03,233 | django.server | "GET /accounts/login/?next=/dashboard/contributor/ HTTP/1.1" 404 6822
[WARNING] 2025-10-26 22:16:22,758 | django.server | "GET /static/accounts/default_avatar.png HTTP/1.1" 404 1895
[WARNING] 2025-10-26 22:23:32,743 | django.server | "GET /static/accounts/default_avatar.png HTTP/1.1" 404 1895
[WARNING] 2025-10-26 22:23:39,643 | django.server | "GET /static/accounts/default_avatar.png HTTP/1.1" 404 1895
[WARNING] 2025-10-26 22:23:41,202 | django.server | "GET /static/accounts/default_avatar.png HTTP/1.1" 404 1895
[WARNING] 2025-10-26 22:23:53,391 | django.request | Not Found: /dashboard/jn/
[WARNING] 2025-10-26 22:23:53,391 | django.server | "GET /dashboard/jn/ HTTP/1.1" 404 6787
[WARNING] 2025-10-26 22:24:06,424 | django.request | Not Found: /j
[WARNING] 2025-10-26 22:24:06,426 | django.server | "GET /j HTTP/1.1" 404 6751
[ERROR] 2025-10-26 22:24:19,275 | django.request | Internal Server Error: /dashboard/contributor/profile/
Traceback (most recent call last):
  File "D:\IT\_OER\gauri oer\oer\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\IT\_OER\gauri oer\oer\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IT\_OER\gauri oer\oer\venv\Lib\site-packages\django\contrib\auth\decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IT\_OER\gauri oer\oer\accounts\views\contributor\contributor_dashboard.py", line 75, in contributor_profile
    return render(request, "403.html", status=403)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IT\_OER\gauri oer\oer\venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IT\_OER\gauri oer\oer\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IT\_OER\gauri oer\oer\venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: 403.html
[ERROR] 2025-10-26 22:24:19,278 | django.server | "GET /dashboard/contributor/profile/ HTTP/1.1" 500 93879
[WARNING] 2025-10-26 22:24:35,242 | django.request | Not Found: /dashboard/contributor/profile/Umme Atiya
[WARNING] 2025-10-26 22:24:35,243 | django.server | "GET /dashboard/contributor/profile/Umme%20Atiya HTTP/1.1" 404 6870
[WARNING] 2025-10-26 22:24:39,004 | django.request | Not Found: /dashboard/contributor/profile/Atiya
[WARNING] 2025-10-26 22:24:39,005 | django.server | "GET /dashboard/contributor/profile/Atiya HTTP/1.1" 404 6853
[WARNING] 2025-10-26 22:24:42,818 | django.request | Not Found: /dashboard/contributor/profile/Umme
[WARNING] 2025-10-26 22:24:42,819 | django.server | "GET /dashboard/contributor/profile/Umme HTTP/1.1" 404 6850
[WARNING] 2025-10-26 22:25:47,246 | django.request | Not Found: /favicon.ico
[WARNING] 2025-10-26 22:25:47,247 | django.server | "GET /favicon.ico HTTP/1.1" 404 6781
[WARNING] 2025-10-26 22:57:00,908 | django.server | "GET /static/frontend/css/style.css HTTP/1.1" 404 1880
[WARNING] 2025-10-26 22:57:01,162 | django.request | Not Found: /favicon.ico
[WARNING] 2025-10-26 22:57:01,168 | django.server | "GET /favicon.ico HTTP/1.1" 404 8745
[WARNING] 2025-10-26 23:50:09,312 | django.server | "GET /static/frontend/css/style.css HTTP/1.1" 404 1880
[WARNING] 2025-10-26 23:50:16,964 | django.request | Not Found: /forum/1/
[WARNING] 2025-10-26 23:50:16,964 | django.server | "GET /forum/1/ HTTP/1.1" 404 6772
[WARNING] 2025-10-26 23:50:18,353 | django.request | Not Found: /favicon.ico
[WARNING] 2025-10-26 23:50:18,354 | django.server | "GET /favicon.ico HTTP/1.1" 404 6781
[WARNING] 2025-10-26 23:50:31,294 | django.request | Not Found: /favicon.ico
[WARNING] 2025-10-26 23:50:31,295 | django.server | "GET /favicon.ico HTTP/1.1" 404 6781
[WARNING] 2025-10-26 23:50:39,002 | django.server | "GET /static/frontend/css/style.css HTTP/1.1" 404 1880
[WARNING] 2025-10-26 23:50:39,089 | django.request | Not Found: /favicon.ico
[WARNING] 2025-10-26 23:50:39,090 | django.server | "GET /favicon.ico HTTP/1.1" 404 6781
[WARNING] 2025-10-26 23:54:17,401 | django.request | Not Found: /favicon.ico
[WARNING] 2025-10-26 23:54:17,402 | django.server | "GET /favicon.ico HTTP/1.1" 404 8745
